src/script/hydra-notify

```diff
@@ -88,6 +127,10 @@ for my $build ($db->resultset('Builds')->search(
     my $buildId = $build->id;
     print STDERR "sending notifications for build ${\$buildId}...\n";
     buildFinished($build);
+
```

Comment:
I don't think this is needed. My understanding is that it's used for the test, and I wrote in the test how one can do it differently, alleviating the need for once.